os/board/common/Kconfig

```diff
@@ -54,3 +54,53 @@ config FLASH_PART_NAME
 
 endif # FLASH_PARTITION
 endif # ARCH_USE_FLASH && (NFILE_DESCRIPTORS != 0)
+
+config ARCH_USE_EXTERNAL_FLASH
+	bool
+	default n
+	---help---
+		This shows that user enables External FLASH to use.
+
+if ARCH_USE_EXTERNAL_FLASH && (NFILE_DESCRIPTORS != 0)
+comment "External Flash Partition Options"
+
+config EXTERNAL_FLASH_PARTITION
```

Comment:
How about using `2nd` instead of `external`? Because almost flash is an external module.